### PR TITLE
RequestLeave Reply Embed

### PR DESCRIPTION
**Pull request by MabroukTBN**
Send the leave requester an embed with their request information and status

### CODE REVIEW

File: Leave/leave_interface.py [hunk 1]
```diff
@@ -66,20 +66,33 @@ async def HandleLeaveReactions(client, payload):
...
+    embed_dict = embed.to_dict()
+    for field in embed_dict["fields"]:
+        if field["name"].lower() == "reason":
+            reason = field["value"]
+        if field["name"].lower() == "leave type":
+            leave_type = field["value"]
+        if field["name"].lower() == "start date":
+            start_date = datetime.datetime.strptime(field["value"], '%Y-%m-%d')
+        if field["name"].lower() == "end date":
+            end_date = datetime.datetime.strptime(field["value"], '%Y-%m-%d')
```
Comment (AbdoHamed-TheBadNomads) [inline Leave/leave_interface.py:92]: i think those should be there own separate functions in the utils script, just like `utils.GetMemberIDFromEmbed(embed)`

File: Leave/leave_interface.py [hunk 2]
```diff
@@ -66,20 +66,33 @@ async def HandleLeaveReactions(client, payload):
...
+
+    replyembed = UI.CreateInformMemberOfLeaveStatusEmbed(status, admin.display_name, reason, leave_type, start_date, end_date)
```
Comment (AbdoHamed-TheBadNomads) [inline Leave/leave_interface.py:94]:
```suggestion
    reply_embed = UI.CreateInformMemberOfLeaveStatusEmbed(status, admin.display_name, reason, leave_type, start_date, end_date)
```

File: UI.py [hunk 3]
```diff
@@ -193,6 +193,33 @@ def CreateLeavesBalancesEmbed(member_id):
...
         return None
 
+def CreateInformMemberOfLeaveStatusEmbed(status, admin_name, reason, leave_type, start_date, end_date):
+    leaveImages = {
+    "Annual"   : os.getenv("Annual_Leave_Link"),
+    "Sick"     : os.getenv("Sick_Leave_Link"),
+    }
+    embed = discord.Embed(
+        title = f'Leave Request Status',
+        description = f'Your request was ' + status.lower(),
```
Comment (AbdoHamed-TheBadNomads) [inline UI.py:203]: i think stating the status in the very beginning of the embed, makes the `embed.add_field(name = "Status", value = status, inline = True)` obsolete.

```suggestion
        description = f'Request No. ' + request_id,
```
would be better, no ? @AntWong-TheBadNomads

File: Leave/leave_interface.py (hunk 2 above)
Comment (MabroukTBN) [reply Leave/leave_interface.py]: Done

File: UI.py (hunk 3 above)
Comment (MabroukTBN) [reply UI.py:203]: I agree. Switched to display request id.

File: Leave/leave_interface.py (hunk 1 above)
Comment (MabroukTBN) [reply Leave/leave_interface.py:92]: Added `GetFieldFromEmbed(embed, field_name)` to *Utilities.py*

**Pull request merged**